src/transformers/models/longt5/modeling_longt5.py

```diff
@@ -1199,6 +1199,11 @@ def forward(
         hidden_states, present_key_value_state = self_attention_outputs[:2]
         attention_outputs = self_attention_outputs[2:]  # Keep self-attention outputs and relative position weights
 
+        # clamp inf values to enable fp16 training
+        if torch.isinf(hidden_states).any():
```

Comment:
Is there any reason not to include `if hidden_states.dtype == torch.float16` in the condition, which is done for `T5`

https://github.com/huggingface/transformers/blob/e1dd784b5c6f2c356835f5200d297e08058eb18c/src/transformers/models/t5/modeling_t5.py#L686

Same question for the other 2 places.

Reply:
No, probably I saw an old version of the code. 😅 
Will change the code to include the `torch.float16`